modules/ding_groups/ding_groups.info

```diff
@@ -14,7 +13,6 @@ dependencies[] = list
 dependencies[] = manualcrop
 dependencies[] = media
 dependencies[] = menu_position
-dependencies[] = metatag_panels
```

Comment:
I don't think the metatag_panels is used in the ding_groups module, can't find it anywhere in the feature exports in the current master.